### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by MatthewJA**
Fixes odc-tools and python-dateutil in setup.py of dea-tools.

### CODE REVIEW

File: Tools/setup.py
Comment (emmaai) [inline Tools/setup.py]: Can you do `pip list | grep 'odc'` and check what exactly was installed on the sandbox or NCI? I did mine.
```
jovyan@jupyter-anotheremma:~$ pip list | grep 'odc'
odc-aio                           0.1.dev674+g2936179
odc-algo                          0.1.dev674+g2936179
odc-apps-cloud                    0.1.dev674+g2936179
odc-apps-dc-tools                 0.1.dev674+g2936179
odc-aws                           0.1.dev674+g2936179
odc-dscache                       0.1.dev674+g2936179
odc-dtools                        0.1.dev674+g2936179
odc-geom                          0.1.dev674+g2936179
odc-index                         0.1.dev674+g2936179
odc-io                            0.1.dev674+g2936179
odc-ppt                           0.1.dev674+g2936179
odc-stats                         0.1.dev674+g2936179
odc-thredds                       0.1.dev674+g2936179
odc-ui                            0.1.dev674+g2936179
```

Comment (emmaai) [inline Tools/setup.py]: this line won't work

Comment (MatthewJA) [reply Tools/setup.py]: oh they're all separate? okey, can do

**Review by emmaai — CHANGES_REQUESTED** [review]

**Comment by MatthewJA** [comment]
That should work now. I've had to modify the pip installation instructions to match odc-tools. The only one we use is odc-ui so I've just included that.

**Comment by robbibt** [comment]
Hey @MatthewJA + @emmaai , can this be merged in?

**Comment by MatthewJA** [comment]
I believe so, just waiting for an approving review.

**Comment by robbibt** [comment]
I can approve if @emmaai is busy, but this is out of my area of expertise so 🤷

**Review by emmaai — APPROVED** [review]
Good job!!

**Comment by emmaai** [comment]
Sorry forgot about this, just approved and merged.

**Pull request merged**
